bundles/com.e1c.v8codestyle.bsl/src/com/e1c/v8codestyle/bsl/check/ExtensionVariablePrefixCheck.java

```diff
@@ -82,8 +86,9 @@ protected void configureCheck(CheckConfigurer builder)
             .issueType(IssueType.CODE_STYLE)
             .extension(new CommonSenseCheckExtension(getCheckId(), BslPlugin.PLUGIN_ID))
             .extension(new AdoptedModuleOwnerExtension())
+            .extension(ModuleTypeFilter.excludeTypes(COMMAND_MODULE, SESSION_MODULE, COMMON_MODULE, MANAGER_MODULE))
             .module()
-            .checkedObjectType(IMPLICIT_VARIABLE, EXPLICIT_VARIABLE, SIMPLE_STATEMENT, DECLARE_STATEMENT);
+            .checkedObjectType(EXPLICIT_VARIABLE, SIMPLE_STATEMENT, DECLARE_STATEMENT);
```

Comment:
```suggestion
            .checkedObjectType(EXPLICIT_VARIABLE, DECLARE_STATEMENT);
```

Вопрос в том, что экспортных переменных не бывает в SIMPLE_STATEMENT - экспортные могут быть только в DECLARE_STATEMENT.